pkgs/tools/misc/ideviceinstaller/default.nix

```diff
@@ -49,7 +49,7 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.gpl2Plus;
     platforms = platforms.unix;
-    maintainers = with maintainers; [ aristid infinisil ];
+    maintainers = [ aristid ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ aristid ];
```

Reply:
Oof this is the second instance of that, fixed!

Reply:
And had to fix another :sweat_smile: